flathub.json

```diff
@@ -1,4 +1,4 @@
 {
-  "only-arches": ["x86_64"],
+  "only-arches": ["x86_64", "aarch64"],
```

Comment:
there is no other arch so the line should be removed.

Reply:
But what if, one day, RISC-V? :)

Reply:
it won't on obsolete 22.08. Nor 23.08. So it won't matter.

It's good to have dreams.

Reply:
OK, that's a great point.